mr/src/main/java/org/apache/iceberg/mr/hive/serde/objectinspector/IcebergTimestampObjectInspector.java

```diff
@@ -80,22 +57,16 @@ public TimestampWritable getPrimitiveWritableObject(Object o) {
 
   @Override
   public Object copyObject(Object o) {
-    if (o == null) {
```

Comment:
Why did we remove this?

Reply:
this null check is handled in the `else` branch